cms/toolbar/toolbar.py

```diff
@@ -97,7 +98,7 @@ def init_toolbar(self, request):
         self.request = request
         self.is_staff = self.request.user.is_staff
         self.edit_mode = self.is_staff and self.request.session.get('cms_edit', False)
-        self.build_mode = self.is_staff and self.request.session.get('cms_build', False)
+        self.build_mode = self.is_staff and request.GET.get(get_cms_setting('CMS_TOOLBAR_URL__BUILD')) != None
```

Comment:
What's the intended behavior?
If ``?build=1`` is in the URL -> switch to build mode and set it in the session
If ``?build=0`` is in the URL -> switch to build mode and set it in the session
If no ``?build`` is in the URL -> remember the session